src/agent/README.md

```diff
@@ -113,3 +113,43 @@ agent control tool, rather than the Kata runtime.
 
 For further details, see the
 [agent control tool documentation](../tools/agent-ctl/README.md#run-the-tool-and-the-agent-in-the-same-environment).
+
+## Agent options
+
+The kata agent has the ability to configure agent options in guest kernel command line at runtime. Currently, the following agent options can be configured: 
+
+| Option | Name | Description  | Type | Default |
+|-|-|:-:|:-:|:-:|
+| `agent.config_file` | `Configuration file` | Allow to configure options through a configuration file from the root filesystem | string | `""` |
+| `agent.container_pipe_size` | `Container pipe sizes` | Allow to configure the stdout or stderr pipe sizes | integer | `0` |
+|`agent.debug_console` | Debug console flag | Allow to connect guest OS running inside hypervisor Connect using `kata-runtime exec <sandbox-id>` | boolean | `false` |
```

Comment:
Sorry, really minor nit that doesn't affect the output, but the inconsistency triggers me:
```suggestion
| `agent.debug_console` | Debug console flag | Allow to connect guest OS running inside hypervisor Connect using `kata-runtime exec <sandbox-id>` | boolean | `false` |
```